packages/url/README.md

```diff
@@ -30,7 +30,7 @@ const newURL = addQueryArgs( 'https://google.com', { q: 'test' } ); // https://g
 
 _Parameters_
 
--   _url_ `?string`: URL to which arguments should be appended. If omitted, only the resulting querystring is returned.
+-   _url_ `[string]`: URL to which arguments should be appended. If omitted, only the resulting querystring is returned.
```

Comment:
I find the `[string]` syntax strange. Where does it come from?

I assume it means "optional parameter with a default value", but that only seems to vaguely align with jsdoc, where `[square brackets]` are used on the parameter or property names to indicate they're optional, not around the type.

To add to the confusion, in TypeScript `[ string ]` would mean a one-tuple of strings, or an array with a single `string` member, e.g. `[ "foo" ]` but never `[]` or `[ "foo", "bar" ]`.